.gitignore

```diff
@@ -0,0 +1,13 @@
+# IntelliJ IDEA
+.idea
+*.iml
+*.ipr
+*.iws
+
+# Gradle
+.gradle
+build/
+
+# Other
```

Comment:
review empty gitignore deletions

Reply:
* twister-lib-java/lib/general/src/main|test/resources/.gitignore -> no evidence of any history in SVN `general/src` came to be after renaming `twister-lib-java/src`, but nothing before move either.